app/components/TextInput/TextInput.jsx

```diff
@@ -147,15 +152,21 @@ export default class TextInput extends React.Component {
 
   componentDidMount() {
     this.adjustHeightIfNeeded()
+
+    const {inputRef} = this.props
+
+    typeof inputRef === 'function' && inputRef(this.base)
```

Comment:
I'm personally not a fan of this type of conditional, as it's not immediately clear the code is branching off. Plus, we use `if (typeof x === 'function')` in numerous places across the application to check whether a prop or callback are defined, so I'd rather keep with that format whenever possible.

Reply:
Fixed